org.sasview.sasview.yml

```diff
@@ -0,0 +1,59 @@
+id: org.sasview.sasview
+runtime: org.kde.Sdk
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+command: sasview
+finish-args:
+  - "--socket=fallback-x11"
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--device=dri"
+  - "--filesystem=~/sasview.log"
+  - "--filesystem=~/.sasview"
+  - "--filesystem=~/.sasmodels"
```

Comment:
use persist or store in XDG compliant locations not in home

Reply:
I agree that they should be in XDG compliant locations, and I will raise an issue with this upstream. However, any change would not happen until the next release. Using persist would mean that a user would not be able to manually update their configuration. Furthermore, SasView supports custom models which are stored in these directories.

Reply:
> Using persist would mean that a user would not be able to manually update their configuration

why would they be  not able to manually update? These folders will be created under `~/.var/app/$app-id/.sasview`.

Reply:
I've just used `--persist=.` so it includes the `sasview.log` file as well. If this is ok, then please mark this conversation as resolved.

Reply:
dumping a file in $HOME like that? that's bad

Reply:
persist does not work with files only directories